### PR TITLE
Homework-01 for TAT.

### PR DESCRIPTION
**Pull request by DmitryZakharenko**

### CODE REVIEW

File: Homework-01/Task-01/src/by/bsu/tat/main/CommandLineParams.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package by.bsu.tat.main;
+
+/**
+ * @author Dmitry Zakharenko
```
Comment (AlexanderSoroka) [inline Homework-01/Task-01/src/by/bsu/tat/main/CommandLineParams.java]: You can combine this comment with the next one

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package by.bsy.tat.main;
+
+import java.util.Scanner;
+
+/**
+ * @author Dmitry Zakharenko
```
Comment (AlexanderSoroka) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: You can combine this with the next one

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ * Main class which contained entry point.
+ * It read input <code>String</code> from command line and print
+ * "Hello, <code>String</code>".
+ * If <code>String</code> is empty than print "Hello, User".
+ */
+public class HelloUserName {
+
+    public static void main(String[] args) {
```
Comment (AlexanderSoroka) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: Missed comment for method main

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        System.out.println("Please, enter your name.");
+        Scanner inputScanner = new Scanner(System.in);
```
Comment (AlexanderSoroka) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: Please, re-read task carefully - it must take name from command line not interact with user via console input

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        String userName = inputScanner.nextLine();
+        if(!userName.isEmpty())
```
Comment (AlexanderSoroka) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: Missed { and }

File: Homework-01/Task-03/src/by/bsu/tat/main/Calculator.java [hunk 6]
```diff
@@ -0,0 +1,63 @@
+package by.bsu.tat.main;
+
+/**
+ *
+ * The class containing methods for some basic numeric operations like as
```
Comment (AlexanderSoroka) [inline Homework-01/Task-03/src/by/bsu/tat/main/Calculator.java]: contains

File: Homework-01/Task-03/src/by/bsu/tat/main/Main.java [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+ * It read input from command line as two numbers and print some basic
+ * operations between it like sum, subtraction, multiplication and division.
+ * The program read only two number coming first. The following numbers are
+ * ignored.
+ *
+ * @author Dmitry Zakharenko
+ */
+public class Main {
+
+    public static void main(String[] args) {
```
Comment (AlexanderSoroka) [inline Homework-01/Task-03/src/by/bsu/tat/main/Main.java]: Missed comment to method main

File: Homework-01/Task-03/src/by/bsu/tat/main/Main.java [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        Scanner inputString = new Scanner(System.in);
+        System.out.println(Helper.START_MESSAGE);
+
+        //try to read and calculate
+        try {
+            String firstValueString = inputString.next();
```
Comment (AlexanderSoroka) [inline Homework-01/Task-03/src/by/bsu/tat/main/Main.java]: Please, re-read task carefully - it must take numbers from command line not interact with user via console input

File: Homework-01/Task-03/src/by/bsu/tat/main/Main.java [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+     * @param secondValue a second number
+     */
+    public static void printResult(double firstValue, double secondValue) {
+        System.out.println(firstValue + " + " + secondValue + " = " +
+                Calculator.sum(firstValue, secondValue));
+        System.out.println(firstValue + " - " + secondValue + " = " +
+                Calculator.subtraction(firstValue, secondValue));
+        System.out.println(firstValue + " * " + secondValue + " = " +
+                Calculator.multiplication(firstValue, secondValue));
+        System.out.println(firstValue + " / " + secondValue + " = " +
```
Comment (AlexanderSoroka) [inline Homework-01/Task-03/src/by/bsu/tat/main/Main.java]: Salary example (NaN) :)

File: Homework-01/Task-01/src/by/bsu/tat/main/CommandLineParams.java (hunk 1 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-01/src/by/bsu/tat/main/CommandLineParams.java]: Fixed.

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java (hunk 2 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: Fixed.

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java (hunk 3 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: Fixed.

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java (hunk 5 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: Fixed.

File: Homework-01/Task-03/src/by/bsu/tat/main/Calculator.java (hunk 6 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-03/src/by/bsu/tat/main/Calculator.java]: Fixed.

File: Homework-01/Task-03/src/by/bsu/tat/main/Main.java (hunk 7 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-03/src/by/bsu/tat/main/Main.java]: Fixed.

File: Homework-01/Task-03/src/by/bsu/tat/main/Main.java (hunk 9 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-03/src/by/bsu/tat/main/Main.java]: I think fixed.

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java (hunk 4 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: hmm, we must take value from parameters of program? From String[args] in main?

File: Homework-01/Task-03/src/by/bsu/tat/main/Main.java (hunk 8 above)
Comment (DmitryZakharenko) [inline Homework-01/Task-03/src/by/bsu/tat/main/Main.java]: hmm, we must take value from parameters of program? From String[args] in main?

File: Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java (hunk 4 above)
Comment (AlexanderSoroka) [inline Homework-01/Task-02/src/by/bsy/tat/main/HelloUserName.java]: right

**Pull request merged**